docker/Dockerfile.dist.ubuntu16.04

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
```

Comment:
`/build-artifacts.sh` use `Docker.ubuntu16.04` to compile and output artifacts.
`/build-docker.sh` use `Docker.dist.ubuntu16.04` and `build-artifacts.sh`'s output (artifacts) to build docker heron images.